internal/display/plugin.go

```diff
@@ -75,9 +77,9 @@ func GetPluginResource(ctx context.Context, name string) (*PluginResource, error
 		return nil, fmt.Errorf("unable to obtain plugin details: %w", err)
 	}
 
-	installedInfo, err := plugin.Get(ctx, config.GlobalConfig.PluginVersions, name)
-	if err != nil {
-		return nil, fmt.Errorf("unable to obtain plugin details: %w", err)
+	version := "local"
```

Comment:
why do we need to default to "local"?

Reply:
shouldn't local plugins be in the version file already?

Reply:
For truly "local" plugins (not just locally built), I figured we may or may not have them in the version file so was a logical safe default?